css/order.css

```diff
@@ -108,16 +108,28 @@
 
 .bnomics-powered-by {
   text-align: center;
-  margin-top: 1.6em;
-  font-size: 0.8em;
+  margin-top: 2.6em;
+  font-size: 0.6em;
   padding: 0;
 }
 
+.bnomics-how-to-pay {
+text-align: center;
+font-size: 12px;
+padding: 0;
+}
+
+.bnomics-how-to-pay a {
+color: #6d6d6d;
```

Comment:
We shouldn't use the color style. Our main focus is to respect the website theme
Actually we should also not use absolute font-size and only use <h1> <h2> etc..

But we are already using font-size is various places (I am not sure why ?). Maybe we could switch to relative font-size (so if we only specify percentage change relative to theme's default font size). @DarrenWestwood can give more detail on this

Reply:
ok, so, color style for the link should not be done, too? because a link would be different. also, we can use em measurements for the fontsize if it's okay.

Reply:
yes. lets use em everywhere. Please attach a screenshot on how it would look as a normal link

Reply:
![image](https://user-images.githubusercontent.com/33615494/66899551-5e6a6d00-f025-11e9-8b09-afcd84d6850d.png)
I already removed the color too.

Reply:
Yes, the hyperlink color is not looking good. Is there a way to force to the default color of the theme instead of picking up the `<a>` tag color

Reply:
i used color:inherit. it seems to be working.
![image](https://user-images.githubusercontent.com/33615494/66918509-b7e49300-f049-11e9-9645-f790ddecc692.png)

Reply:
yes this looks good. @DarrenWestwood  please review the pull